user/plan9port/files/acme

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec 9 acme
```

Comment:
does this need "$@" too? (and i'm not sure if it accepts files passed, for perhaps %U or whatnot to the desktop entry Exec (see the spec))